tests/tool/Integration.Tests/IntegrationScenarios/AspNetSample/csharp/Original/MyControllerLibrary/MyControllerLibrary.csproj

```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Unit tests are probably sufficient. I think you're planning to remove these pre-merge, though.

Reply:
Of note: switching from 2 projects with a solution file back to a single project integration test scenario changes the runtime of that integration test from an average of 2min to an average of 1min on my machine.